src/main/webapp/new/app/play/play.component.ts

```diff
@@ -40,23 +44,57 @@ import { DiceComponent } from './dice/dice.component';
     providers: [
         PlayService,
         SelectService,
+        ActionService,
         MarkingService,
         TemplatesService
     ],
     inputs: ['game']
 })
 
-export class PlayComponent implements OnInit {
+export class PlayComponent implements OnInit, OnDestroy {
     game: Game;
+    availableActions: AvailableActions;
 
     templatesLoaded: boolean = false;
 
-    constructor(private _templates: TemplatesService) { }
+    constructor(
+        private _notification: NotificationService,
+        private _authUser: AuthUserService,
+        private _action: ActionService,
+        private _templates: TemplatesService) { }
 
     ngOnInit() {
+        this.availableActions = this.game.getCurrentPlayer(this._authUser.get()).availableActions;
```

Comment:
you use this variable below calling 'this._checkIfImmediateAndExecute();', move it down